src/components/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import ls from 'local-storage';
+import { NavLink } from 'react-router-dom';
+
+import { Button } from '@/shared';
+
+import './RegisterForm.scss';
+
+const CN = 'register-form';
+
+class RegisterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: ''
+      // password: '',
+      // name: ''
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    this.setState({
+      email: ls.get('emailToRegister') || ''
+    });
+  }
+
+  saveToLocalStore() {
+
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const form = e.target;
+    const data = new FormData(form);
+
+    fetch('/api/form-submit-url', {
+      method: 'POST',
+      body: data
+    });
+  }
+
+  render() {
+    const { email } = this.state;
+
+    return (
+      <div className={`${CN}`} onSubmit={this.handleSubmit}>
+        <span className={`${CN}__label`}>create an account</span>
+        <div className={`${CN}__wrapper`}>
+          <form className={`${CN}__personal-information ${CN}__wrapper__personal-information`}>
+
+            <h3>personal information</h3>
+            <div className={`${CN}__field`}>
+              <label className={`${CN}__field__label`}>first name*</label>
+              <input className={`${CN}__field__input`} type="text" required />
+            </div>
+
+            <div className={`${CN}__field`}>
+              <label className={`${CN}__field__label`}>last name*</label>
+              <input className={`${CN}__field__input`} type="text" required />
+            </div>
+
+            <div className={`${CN}__field`}>
+              <label className={`${CN}__field__label`}>e-mail*</label>
+              <input className={`${CN}__field__input`} type="email" id="email" value={email} required />
```

Comment:
When there is more than 3 properties, it is better to start each property in a new row to make it more readable, like this:
<input
 className={`${CN}__field__input`}
 type="email"
 id="email"
 value={email}
 required
/>